index.js

```diff
@@ -67,13 +67,29 @@ class ExecutorRouter extends Executor {
      * @param {String} config.token         JWT to act on behalf of the build
      * @return {Promise}
      */
-    start(config) {
+    _start(config) {
```

Comment:
Where do these functions get called? Is there anywhere that cares about `start` changing to `_start`?